python/cudf/cudf/tests/test_groupby.py

```diff
@@ -788,25 +801,32 @@ def test_groupby_index_type():
     assert isinstance(res.index, cudf.core.index.StringIndex)
 
 
+@pytest.mark.xfail()
```

Comment:
We can't just xfail this, we need to fix the underlying problem.

Reply:
agreed. Quantile weighting is likely the culprit for all of `groupby`. I suspected it was new `datetime` features but tests say that doesn't matter and the new `quantile` math in `pandas 0.25+` knocks everything out of alignment. It should be pretty easy to get it back in parity, their code is pretty clean and ours is sparse.